tools/eksDistroBuildToolingOpsTools/pkg/consumerUpdater/bottlerocket.go

```diff
@@ -1,13 +1,24 @@
 package consumerUpdater
 
 import (
+	"bytes"
 	"fmt"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/logger"
+	"os"
+	"path/filepath"
+	"regexp"
 
 	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/eksDistroRelease"
 )
 
 const (
-	bottlerocketName = "Bottlerocket"
+	bottlerocketName    = "Bottlerocket"
+	versionRegex        = `[0-9]+\.[0-9]+\.[0-9]+`
```

Comment:
This will be a very consistent pattern do we want to move it to pkg/constants.go.

Reply:
done